pkg/ui/src/views/cluster/containers/nodeGraphs/summaryBar.tsx

```diff
@@ -61,7 +61,7 @@ export default function(props: ClusterSummaryProps) {
   // Capacity math used in the summary status section.
   const { capacityAvailable, capacityUsed } = props.nodesSummary.nodeSums;
   const usableCapacity = capacityAvailable + capacityUsed;
-  const capacityPercent = usableCapacity !== 0 ? (capacityUsed / usableCapacity) : 1;
+  const capacityPercent = usableCapacity !== 0 ? (capacityUsed / usableCapacity) : "-";
```

Comment:
yeah, failing typechecker because this is now sometimes a number and sometimes a string, and the `format` function below expects a number…

Reply:
I think if you use `undefined` or `null` here instead of `"-"`, it'll be turned into a `"-"` downstream by `computeValue`: https://github.com/cockroachdb/cockroach/blob/07e2b77d390874e057ec99758c33167f410144c7/pkg/ui/src/views/shared/components/summaryBar/index.tsx#L42

And pass the typechecker

Reply:
What's the point of using Javascript if you don't get to benefit from its ridiculously loose typing? :p

Using `null` worked great, though. Cleaner than what I was going to do (store an appropriately `d3.format()`ted string in `capacityPercent` and switch from using a `SummaryStat` to a `SummaryValue`

Reply:
Lol yeah. Funny thing is, `number | string` is actually a valid TypeScript type; the downstream stuff just wasn't geared to use that… Glad the `null` solution worked.